src/Chirp.Infrastructure/AuthorValidation.cs

```diff
@@ -2,8 +2,10 @@ namespace Chirp.Infrastructure;
 
 public class AuthorValidator : AbstractValidator<Author>
 {
+    // A class to validate the instances of the Author entity before they are stored in the database.
     public AuthorValidator()
     {
+        // Set up the rules that a valid Author entity must follow.
         RuleFor(author => author.AuthorId).NotNull();
         RuleFor(author => author.Name).NotEmpty();
         RuleFor(author => author.Email).NotEmpty();
```

Comment:
Not relevant for this issue, but I though if email check should be an email check, and not just NotEmpty?

Reply:
Totalt enig, skal jeg lige ændre det og comitte? Eller skal den i sit eget issue først?

Reply:
Googlede hurtigt og fluent validation har vist en funktion der tjekker om den inderholder @, så ja og nej @Maquidos

Reply:
Man kan gøre noget ala:

RuleFor(customer => customer.Email).EmailAddress();